### PR TITLE
Auto-Approve Dependabot

### PR DESCRIPTION
**Pull request by philsturgeon**
Automatically approve PRs from Dependabot so we don't have to.

This is a bit less pretty than it could be, due to an error with [auto-approve](https://github.com/hmarr/auto-approve-action/issues/8). I'm following the issue and will clean this up later. It's fine.

### CODE REVIEW

File: .github/workflows/auto-approve.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: kjvalencik/github-actions/approve@master
+        if: github.event_name == 'pull_request' && (github.actor == 'dependabot-preview[bot]' || github.actor == "dependabot[bot]')
```
Comment (P0lip) [inline .github/workflows/auto-approve.yml]:
```suggestion
        if: github.event_name == 'pull_request' && (github.actor == 'dependabot-preview[bot]' || github.actor == 'dependabot[bot]')
```
Syntax error :sweat_smile: 
image

Comment (philsturgeon) [reply .github/workflows/auto-approve.yml]: Just pushed the fix!

File: install.sh [hunk 2]
```diff
@@ -1,35 +0,0 @@
-#!/bin/sh
```
Comment (P0lip) [inline install.sh]: Did you forget to commit the file? I  see it's removed, but not added anywhere

Comment (philsturgeon) [reply install.sh]: good catch! I was rushing.

**Comment by hmarr** [comment]
FYI, I rebuilt the [auto-approve action](https://github.com/hmarr/auto-approve-action) for the new version of GitHub Actions (released as v2.0.0), so that may fix the issues you were having. Sorry it was broken for a little while!

**Pull request merged**